### PR TITLE
Optimized patching script

### PR DESCRIPTION
**Pull request by ckogant**

### CODE REVIEW

**Comment by psairam** [comment]
@ckogant, also handle copying all files from hypervisor to image with single command in add_files function

File: image-patching/stopgap-script/nuage_overcloud_full_patch.py [hunk 1]
```diff
@@ -157,15 +157,10 @@ def add_files(image, version, workingDir):
         virt_customize(
             '"mkdir -p /etc/puppet/modules/nuage/manifests/13_files" -a %s --memsize %s --selinux-relabel' % (
             image, VIRT_CUSTOMIZE_MEMSIZE))
-        virt_copy('%s %s/13_files/neutron_init.pp /etc/puppet/modules/nuage/manifests/13_files' % (image, workingDir))
-        virt_copy('%s %s/13_files/conductor.pp /etc/puppet/modules/nuage/manifests/13_files' % (image, workingDir))
+        virt_copy('%s %s/13_files/* /etc/puppet/modules/nuage/manifests/13_files' % (image, workingDir))
         virt_customize(
-            '"cp /etc/puppet/modules/nuage/manifests/13_files/neutron_init.pp /etc/puppet/modules/neutron/manifests/init.pp" -a %s --memsize %s --selinux-relabel' % (
+            '"cp /etc/puppet/modules/nuage/manifests/13_files/* /etc/puppet/modules/neutron/manifests" -a %s --memsize %s --selinux-relabel' % (
```
[the remark this answers is not in the document]
Comment (ckogant) [reply image-patching/stopgap-script/nuage_overcloud_full_patch.py]: done

File: image-patching/stopgap-script/nuage_overcloud_full_patch.py [hunk 2]
```diff
@@ -211,15 +199,7 @@ def create_repo_file(reponame, repoUrl, image):
...
-              'touch /etc/yum.repos.d/nuage.repo \n'
-              'echo "[Nuage]" >> /etc/yum.repos.d/nuage.repo \n'
-              'echo "name=%s" >> /etc/yum.repos.d/nuage.repo \n'
-              'echo "baseurl=%s" >> /etc/yum.repos.d/nuage.repo \n'
-              'echo "enabled = 1" >> /etc/yum.repos.d/nuage.repo \n'
-              'echo "gpgcheck = 0" >> /etc/yum.repos.d/nuage.repo \n'
-              'EOT' % (reponame, repoUrl)])
+def delete_repo_file(image):
     virt_customize('"rm -f /etc/yum.repos.d/nuage.repo" -a %s --selinux-relabel' % (image))
     cmds_run(['rm -f create_repo'])
```
[the remark this answers is not in the document]
Comment (ckogant) [reply image-patching/stopgap-script/nuage_overcloud_full_patch.py]: done

File: image-patching/stopgap-script/nuage_overcloud_full_patch.py [hunk 3]
```diff
@@ -154,18 +154,18 @@ def uninstall_packages(image, version):
 def add_files(image, version, workingDir):
     version = int(version)
     if version == 13:
+        cmds_run(['cat <<EOT > version_13 \n'
+        'mkdir -p /etc/puppet/modules/nova/manifests/patch \n'
```
[the remark this answers is not in the document]
Comment (ckogant) [reply image-patching/stopgap-script/nuage_overcloud_full_patch.py]: done

**Comment by ckogant** [comment]
These changes are already part of https://github.com/nuagenetworks/nuage-ospdirector/pull/41

**Pull request closed without merging**